libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-context.ts

```diff
@@ -1,10 +1,21 @@
 import { SkyFilterBarFilterValue } from './filter-bar-filter-value';
 
+/**
+ * The context object that is provided to a filter modal.
+ */
 export class SkyFilterBarFilterModalContext {
+  /**
+   * The name of the filter for displaying in the modal title.
+   */
   public filterName: string;
-
+  /**
+   * The value of the filter.
+   */
   public filterValue?: SkyFilterBarFilterValue;
-
+  /**
+   * An untyped property that can track any config information relevant to the filter modal
+   * that existing options do not include.
```

Comment:
```suggestion
   * that isn't provided through other properties.
```
I'm not completely sure if this is better, but I was initially confused about what we meant by "that existing options do not include" so I played around with the text a bit. Hopefully this edit and the edit on the previous line make things clearer, but please just disregard the suggestions if not.

Reply:
Same as above

Reply:
Same question about whether we should leave this as is because it's completely clear here and in data manager docs or if we should make edits here and then follow up with edits to the data manager docs separately?